docs/docs/en/development/api-standard.md

```diff
@@ -71,20 +71,28 @@ Use URI to locate the resource, use DELETE to indicate delete.
 + delete an `alert-group`
 ```
 Method: DELETE
-/api/dolphinscheduler/alter-groups/{alterGroupId}
+/dolphinscheduler/alter-groups/{alterGroupId}
 ```
 
 + batch deletion: batch delete the id array，we should use POST. **（Do not use the DELETE method, because the body of the DELETE request has no semantic meaning, and it is possible that some gateways, proxies, and firewalls will directly strip off the request body after receiving the DELETE request.）**
 ```
 Method: POST
-/api/dolphinscheduler/alter-groups/batch-delete
+/dolphinscheduler/alter-groups/batch-delete
 ```
 
-### ⑤ Others
+### ⑤ Partial Modifications -PATCH
+Use URI to locate the resource, use PATCH to partial modifications.
+
+```
+Method: PATCH
+/dolphinscheduler/alter-groups/{alterGroupId}
```

Comment:
Since there is no actual interface requested by ```PATCH```. I temporarily use a nonexistent interface as an example.

Reply:
I do not know before you mention it